### PR TITLE
Container subdir

### PR DESCRIPTION
**Pull request by joskore**
# Purpose

# Bugfix or Enhancement:

Directory structure has superfluous elements. #67 for context

- [X] Create a directory for holding all container configs separate from the main application
- [X] Move all containers to use this new directory
- [X] Remove superfluous /config from all containers and use bare subdir instead
- [X] Allow user to set a configuration sub-directory to append to config volume mappings. Default to /config

# Requirements

Please ensure your pull request adheres to the following guidelines:

- [X] I have read and agree to the [Contributor Covenant Code of Conduct](https://github.com/joskore/media-docker/blob/master/.github/CODE_OF_CONDUCT.md).
- [X] I have confirmed that all changes adhere to the [Contributing Guidelines](https://github.com/joskore/media-docker/blob/master/.github/CONTRIBUTING.md).
- [X] I have confirmed that all defined tests pass prior to opening this pull request.

Thanks for contributing!

### CODE REVIEW

File: .containers/headphones/headphones.yaml [hunk 1]
```diff
@@ -3,7 +3,7 @@ services:
   headphones:
     container_name: headphones
     volumes:
-      - ${BASE_DIR}/headphones/config:/config
+      - ${CONTAINER_DIR}/headphones/config:/config
```
Comment (powerdude) [inline .containers/headphones/headphones.yaml]: missed one

File: .containers/heimdall/heimdall.yaml [hunk 2]
```diff
@@ -3,7 +3,7 @@ services:
   heimdall:
     container_name: heimdall
     volumes:
-      - ${BASE_DIR}/heimdall/config:/config
+      - ${CONTAINER_DIR}/heimdall/config:/config
```
Comment (powerdude) [inline .containers/heimdall/heimdall.yaml]: missed one

File: .containers/htpcmanager/htpcmanager.yaml [hunk 3]
```diff
@@ -3,7 +3,7 @@ services:
   htpcmanager:
     container_name: htpcmanager
     volumes:
-      - ${BASE_DIR}/htpcmanager/config:/config
+      - ${CONTAINER_DIR}/htpcmanager/config:/config
```
Comment (powerdude) [inline .containers/htpcmanager/htpcmanager.yaml]: missed one

File: .containers/tautulli/tautulli.yaml [hunk 4]
```diff
@@ -3,8 +3,8 @@ services:
   tautulli:
     container_name: tautulli
     volumes:
-      - ${BASE_DIR}/tautulli/config:/config
-      - ${BASE_DIR}/plex/config/Library/Application\ Support/Plex\ Media\ Server/Logs:/logs:ro
+      - ${CONTAINER_DIR}/tautulli:/config
+      - ${CONTAINER_DIR}/plex/config/Library/Application\ Support/Plex\ Media\ Server/Logs:/logs:ro
```
Comment (powerdude) [inline .containers/tautulli/tautulli.yaml]: /plex/config should just be /plex

Comment (joskore) [reply .containers/tautulli/tautulli.yaml]: hey, thanks for all of these comments! this is PR is not completed, and things are still changing here.

Comment (powerdude) [reply .containers/tautulli/tautulli.yaml]: ok.  :)

Comment (joskore) [reply .containers/tautulli/tautulli.yaml]: it's a workflow problem of mine, probably - i like to let travis run builds as i make changes. 

assuming these checks pass correctly, i think i've covered what you pointed out above as well as covered each of the container config sub-directories and the process behind it. if you want to review, i'd appreciate the feedback :)

**Comment by joskore** [comment]
@powerdude the four you called out above were corrected in a later commit in this PR than where you were reviewing

**Pull request merged**